lib/node_modules/@stdlib/array/complex64/lib/main.js

```diff
@@ -917,6 +917,56 @@ setReadOnly( Complex64Array.prototype, 'get', function get( idx ) {
 	return new Complex64( buf[ idx ], buf[ idx+1 ] );
 });
 
+/**
+* Tests whether all elements in a array pass a test implemented by a predicate function.
+*
+* @name every
+* @memberof Complex64Array.prototype
+* @type {Function}
+* @param {Function} [predicate] - predicate - test function
+* @param {*} [thisArg] - context
+* @throws {TypeError} `this` must be a complex number array
+* @throws {TypeError} first argument must be a function
+* @returns {Boolean} - boolean indicating whether all elements pass a test
+*
+* @example
+* var arr = new Complex64Array( 3 );
+* var realf = require( '@stdlib/complex/realf' );
+* var imagf = require( '@stdlib/complex/imagf' );
```

Comment:
```suggestion
* var realf = require( '@stdlib/complex/realf' );
* var imagf = require( '@stdlib/complex/imagf' );
*
* var arr = new Complex64Array( 3 );
```

`require` statements always precede logic.